src/schema/artist/index.js

```diff
@@ -33,6 +33,8 @@ import { GraphQLObjectType, GraphQLBoolean, GraphQLString, GraphQLNonNull, Graph
 import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
 import { parseRelayOptions } from "lib/helpers"
 
+const whitelistHighlightedAuctionResults = require("../../lib/auction_result_artists.json").artists
```

Comment:
Minor but some of these names could be made more consistent. There's "auction_result_artists," "whitelistHighlightedAuctionResults," and "recordsTrusted." E.g., could they all use "records trusted...?"

Reply:
Also I'd add a comment here about the context/rationale for this hard-coded list.